opentok/opentok-meet
# OPENTOK-47234: enable simulcast screenshare by default

#### What is this PR doing?
We'd like to enable simulcastScreenshare on meet. This will not include an UI component as we currently don't allow users to disable simulcast.

#### How should this be manually tested?
On the local version of meet use an API key that supports simulcast or add `_enableSimulcast: true,` below the new `scalableScreenshare` properties. Verify that multiple video streams are being sent for screenshare on webrtc-internals.

#### What are the relevant tickets?
https://jira.vonage.com/browse/OPENTOK-47234
